test/metabase/api/dashboard_test.clj

```diff
@@ -543,6 +543,28 @@
        (Dashboard dashboard-id)])))
 
 
+(expect (merge dashboard-defaults
+               {:name       "Test Dashboard - Copy"
+                :creator_id (user->id :rasta)})
+        (tt/with-temp Dashboard [{dashboard-id :id} {:name "Test Dashboard"}]
+          (-> ((user->client :rasta) :post 200 (format "dashboard/%d/copy" dashboard-id))
+              (dissoc :id)
+              (update :created_at boolean)
+              (update :updated_at boolean))))
```

Comment:
You can use `metabase.test.util/boolean-ids-and-timestamps` instead of the above three lines, same with line 563-565 as well.